src/uppy-react/bridge/ReactDashboardPlugin.js

```diff
@@ -0,0 +1,31 @@
+const DashboardPlugin = require('../../plugins/Dashboard')
+
+/**
+ * A version of the Dashboard plugin that externalises the modal open state
+ * management, allowing the React component wrapper to open and close it at
+ * will.
+ */
+class ReactDashboardPlugin extends DashboardPlugin {
+  constructor (core, opts) {
```

Comment:
I’m a little worried about doing this. Could we maybe add onRequestClose/hideModal callback into the Dashboard plugin itself? Otherwise its third-level of class nesting, which is not that bad, but starting to feel like a dangerous territory :)